upgrade_tests/test_existance_relations/cli/test_contentviews.py

```diff
@@ -33,7 +32,6 @@
 
 
 # Tests
-@pytest_skip_if_bug_open('bugzilla', 1461026)
```

Comment:
Though WONTFIX we can keep the test
The issue was only with Docker repositories which are now gone